nowplaying/options.py

```diff
@@ -0,0 +1,72 @@
+import configargparse
+
+
+class Options(object):
+    """Contain all hardcoded and loaded from configargparse options."""
+
+    """How many seconds the main daemon loop sleeps."""
+    sleepSeconds = 1
+
+    """Default socket of 2 minutes, to prevent endless hangs on HTTP requests."""
+    socketDefaultTimeout = 120
+
+    def __init__(self):
```

Comment:
Most stuff in here is still configured to run out of the box to aid with (admittedly non external user friendly) dev work.

To really address #11 this will change but we need to build out the container dev env so there is a dev env that also works w/o the RaBe env.